OpenRA.Mods.Common/Util.cs

```diff
@@ -198,15 +198,15 @@ public static IEnumerable<CPos> RandomWalk(CPos p, MersenneTwister r)
 			}
 		}
 
-		public static int RandomDelay(World world, int[] range)
+		public static int RandomDelay(MersenneTwister random, int[] range)
```

Comment:
This method is no longer used for just random delays, so as the PR is now touching this can we please rename it to something like `RandomInRange`?

Reply:
Can we please also split these changes into their own commit before the `WithResourceAnimation` changes?

Reply:
Done.